tests/phpunit/tests/rest-api.php

```diff
@@ -1284,6 +1284,37 @@ public function _dp_rest_filter_response_by_context() {
 				),
 				array( 'additional' => array( 'a' => '1' ) ),
 			),
+			'pattern properties'                  => array(
+				array(
+					'$schema'              => 'http://json-schema.org/draft-04/schema#',
+					'type'                 => 'object',
+					'properties'           => array(
+						'a' => array(
+							'type'    => 'string',
+							'context' => array( 'view', 'edit' ),
+						),
+					),
+					'patternProperties'    => array(
+						'[0-9]' => array(
```

Comment:
Can we add another pattern property entry with a context of just `edit` to demonstrate that the property get's removed?